### PR TITLE
Remove unused package, add missing types

### PR DESCRIPTION
**Pull request by matijs**
Add missing types for `node-gettext`.
Remove `gettext-extractor` as it's no longer used. Dependabot should automatically close its open PR for this as well.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -69,6 +69,7 @@
         "@types/jest": "^24.9.1",
         "@types/lodash.debounce": "^4.0.6",
         "@types/node": "^12.12.26",
+        "@types/node-gettext": "^2.0.1",
```
Comment (ChrisTerBeke) [inline package.json]: I think node-gettext and gettext-parser are also not needed anymore (they are for the same functionality as gettext-extractor was used for).

Comment (matijs) [reply package.json]: i18n.tsx uses it…

Comment (matijs) [reply package.json]: That also uses gettext-parser by the way…

Comment (ChrisTerBeke) [reply package.json]: ah right, we should leave those in then (and only remove gettext-extractor).

**Pull request merged**